x-pack/plugins/security/public/management/roles/edit_role/privileges/es/elasticsearch_privileges.test.tsx

```diff
@@ -75,13 +77,26 @@ test('it renders remote index privileges section when `canUseRemoteIndices` is e
   expect(wrapper.find('IndexPrivileges[indexType="remote_indices"]')).toHaveLength(1);
 });
 
+test('it does not render remote cluster privileges section by default', () => {
+  const wrapper = shallowWithIntl(<ElasticsearchPrivileges {...getProps()} />);
+  expect(wrapper.find('RemoteClusterPrivileges')).toHaveLength(0);
+});
+
+test('it renders remote index privileges section when `canUseRemoteClusters` is enabled', () => {
+  const wrapper = shallowWithIntl(<ElasticsearchPrivileges {...getProps()} canUseRemoteClusters />);
+  expect(wrapper.find('RemoteClusterPrivileges')).toHaveLength(1);
+});
+
 test('it renders fields as disabled when not editable', () => {
-  const wrapper = shallowWithIntl(<ElasticsearchPrivileges {...getProps()} editable={false} />);
+  const wrapper = shallowWithIntl(
```

Comment:
Question: I was trying to manually test this, do you know which cluster privileges a user needs to see these? I tried `read_security` but that seems to hide those sections.

Reply:
Could you try with `manage_security`? Also license needs to be [platinumOrBetter](https://github.com/elastic/kibana/pull/182377/files#diff-9d7c242087ee8b12e417aa6db7001ee5722c261d118d296fa2280f9fc31fcad7R124) and [remote_clusters](https://github.com/elastic/kibana/pull/182377/files#diff-9d7c242087ee8b12e417aa6db7001ee5722c261d118d296fa2280f9fc31fcad7R124) feature set

Reply:
`read_security` should be sufficient to see a full read-only version of this form, but Elena is right about the other requirements.

Reply:
Created a separate issue for this https://github.com/elastic/kibana/issues/182847